src/tokens/tokens.utils.ts

```diff
@@ -95,9 +97,12 @@ export async function getVaultTokens(
       });
     }
   }
-  const sdk = await chain.getSdk();
-  const tokenInfo = await sdk.tokens.loadToken(vaultToken);
-  return Promise.all([toBalance(tokenInfo, balance, currency)]);
+
+  const tokenInfo = await getFullToken(chain, vaultToken);
+
+  if (!tokenInfo) return [];
```

Comment:
hmmm, not sure what is worse here. for this case in particular we are needing the token info the for `decimals` field and without it the impact is an improper balance... i wonder if its better to just throw an error

this only happens with tokens (if we assume a standard 18 decimals) that have less than that which i think is atm maybe 3 tokens total we encounter

Reply:
Error sgtm